src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -413,6 +415,19 @@ internal void WorkerInitResponse(GrpcEvent initEvent)
             _workerInitTask.TrySetResult(true);
         }
 
+        private void LogWorkerMetadata(WorkerMetadata workerMetadata)
+        {
+            if (workerMetadata == null)
+            {
+                return;
+            }
+
+            workerMetadata.UpdateWorkerMetadata(_workerConfig);
+            var workerMetadataString = workerMetadata.ToString();
+            _metricsLogger.LogEvent(MetricEventNames.WorkerMetadata, functionName: null, workerMetadataString);
+            _workerChannelLogger.LogDebug("Worker metadata: {workerMetadata}", workerMetadataString);
```

Comment:
This may not be applicable for the scenario here but wanted to call out that for one of my previous PRs - [refer](https://github.com/Azure/azure-functions-host/pull/8796#discussion_r987394851), I got to know that adding log lines here has perf impact. So, we can double check if this log is really needed here and not affecting the perf.

Reply:
This logging was existing code, which I moved to a private method now. So this PR should not make a perf impact as such.

But I get your point. We can definitely discuss removing this logging entry as we have this information getting logged to functions metrics as well. Thoughts @liliankasem ?